### PR TITLE
add module to merge assets

### PR DESCRIPTION
**Pull request by pbivic**

### CODE REVIEW

**Review by jeromefellus-sekoia — APPROVED** [review]

File: Sekoia.io/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+## 2024-10-08 - 2.64.5
+
+### Changed
```
Comment (goudyj) [inline Sekoia.io/CHANGELOG.md:11]:
```suggestion
### Added
```

File: Sekoia.io/action_assets_merge.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{
+  "name": "MergeAssets",
+  "description": "Merge a list of asset into a targeted asset",
```
Comment (goudyj) [inline Sekoia.io/action_assets_merge.json:3]:
```suggestion
  "description": "Merge a list of assets into a targeted asset",
```

File: Sekoia.io/action_assets_merge.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  "uuid": "86fc9d59-3c54-56f3-8266-1c9d80030a48",
+  "docker_parameters": "MergeAssets",
```
Comment (goudyj) [inline Sekoia.io/action_assets_merge.json:5]:
```suggestion
  "docker_parameters": "merge-assets",
```

File: Sekoia.io/main.py [hunk 4]
```diff
@@ -79,6 +80,7 @@
     module.register(ReportsGetReport, "get-reports/{uuid}")
     module.register(GetEventFieldCommonValues, "get-event-field-common-values")
     module.register(GetEvents, "get-events")
+    module.register(MergeAssets, "MergeAssets")
```
Comment (goudyj) [inline Sekoia.io/main.py:83]:
```suggestion
    module.register(MergeAssets, "merge-assets")
```

Comment (pbivic) [reply Sekoia.io/main.py:83]: Btw, this names is created automatically when using the sdk

**Comment by pbivic** [comment]
Hello @squioc :)
Same question as [here](https://github.com/SEKOIA-IO/automation-library/pull/1135):
 How can I test it on the test platform ? Thanks

**Comment by goudyj** [comment]
As soon as it is merged you can try it on test @pbivic

**Pull request merged**